tests/tests.py

```diff
@@ -211,7 +228,7 @@ def tearDown(self):
     # now we check for existence of the test file
     # if it does not exist then we create it
     file_name = "test_profile.bedGraph.gz"
-    file_path = "./" + file_name
+    file_path = os.getcwd() + '/' + file_name
```

Comment:
for file paths it would be more portable to use `os.path.sep` (which is '/' on linux but '\\' on windows) or `os.path.join(os.getcwd(), file_name)`